fineract-provider/src/test/resources/features/infrastructure/infrastructure.core.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Core Infrastructure
+
+  # TODO: do we really have to test this?
+  @infrastructure @ignore
```

Comment:
It's essentially verifying that if somebody disables Liquibase migrations, the database won't be touched. I think it's necesssary.

Reply:
Done